app/pages/VersionInfoPage.scala

```diff
@@ -27,8 +26,7 @@ case object VersionInfoPage extends QuestionPage[VersionInfo] {
 
   override def toString: String = "versionInfo"
 
-  override def route(waypoints: Waypoints): Call =
-    routes.IndexController.onPageLoad
+  override def route(waypoints: Waypoints): Call = Call("GET", "")
```

Comment:
Does this Call need a real URL target as second argument?

Reply:
No it doesn't. It isn't used. Just needs to have something there or get compile error.